epics.go

```diff
@@ -59,12 +59,22 @@ func (e Epic) String() string {
 // GitLab API docs: https://docs.gitlab.com/ee/api/epics.html#list-epics-for-a-group
 type ListGroupEpicsOptions struct {
 	ListOptions
-	State    *string `url:"state,omitempty" json:"state,omitempty"`
-	Labels   Labels  `url:"labels,comma,omitempty" json:"labels,omitempty"`
-	AuthorID *int    `url:"author_id,omitempty" json:"author_id,omitempty"`
-	OrderBy  *string `url:"order_by,omitempty" json:"order_by,omitempty"`
-	Sort     *string `url:"sort,omitempty" json:"sort,omitempty"`
-	Search   *string `url:"search,omitempty" json:"search,omitempty"`
+	State            *string    `url:"state,omitempty" json:"state,omitempty"`
+	Labels           Labels     `url:"labels,comma,omitempty" json:"labels,omitempty"`
+	WithLabelDetails *bool      `url:"with_labels_details,omitempty" json:"with_labels_details,omitempty"`
+	Milestone        *string    `url:"milestone,omitempty" json:"milestone,omitempty"`
+	Scope            *string    `url:"scope,omitempty" json:"scope,omitempty"`
+	AuthorID         *int       `url:"author_id,omitempty" json:"author_id,omitempty"`
+	MyReactionEmoji  *string    `url:"my_reaction_emoji,omitempty" json:"my_reaction_emoji,omitempty"`
+	IIDs             []int      `url:"iids[],omitempty" json:"iids,omitempty"`
+	OrderBy          *string    `url:"order_by,omitempty" json:"order_by,omitempty"`
+	Sort             *string    `url:"sort,omitempty" json:"sort,omitempty"`
+	Search           *string    `url:"search,omitempty" json:"search,omitempty"`
+	CreatedAfter     *time.Time `url:"created_after,omitempty" json:"created_after,omitempty"`
+	CreatedBefore    *time.Time `url:"created_before,omitempty" json:"created_before,omitempty"`
+	UpdatedAfter     *time.Time `url:"updated_after,omitempty" json:"updated_after,omitempty"`
+	UpdatedBefore    *time.Time `url:"updated_before,omitempty" json:"updated_before,omitempty"`
+	Confidential     *bool      `url:"confidential,omitempty" json:"confidential,omitempty"`
```

Comment:
I don't see `Milestone`, `Scope`, IIDs` and `Confidential` in the docs. Are you sure that are valid options?

And if we want to make this list complete, we probably need to add `include_ancestor_groups` and `include_descendant_groups` as well.

Last but not least, I try to keep the ordering inline with the ordering in de docs. So please make so that is fixed. Thanks!

Reply:
Sorry, I just rushed it. I'll commit a fix for those issues today.